src/drivers/rc_epgsql.erl

```diff
@@ -0,0 +1,55 @@
+-module (rc_epgsql).
+
+-behaviour (gen_server).
+
+-export ([start_link/1]).
+-export ([init/1, handle_info/2, handle_cast/2, handle_call/3, terminate/2]).
+
+start_link(Args) ->
+  gen_server:start_link({local, ?MODULE}, ?MODULE, Args, []).
+
+init(Args) ->
+  self() ! connect,
+  process_flag(trap_exit, true),
+  {ok, #{retries => 0, connection_args => Args}}.
```

Comment:
any particular reason why you prefer to have connection_args as a nested map? seems simpler to have a single flat state map